Task/Task.2/main.js

```diff
@@ -0,0 +1,31 @@
+let myName = 'Dmytro';
+let mySurname = 'Fil';
+let myCity = 'Kyiv';
+
+console.log(myName, mySurname, myCity)
+
+let someNumber;
+let Num1 = 2,
+    Num2 = 4,
+    Num3 = 6,
+    Num4 = 8;
+
+console.log(Num1, Num2, Num3, Num4);
+
+someNumber = ((Num1 + Num2 + Num3 + Num4) / 4);
```

Comment:
Try to give meaningful names for variables. `someNumber` isn't clear that the it will hold
For example `averageNumberOfFour` will be much more understandable